.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "node": true,
+        "es6": true
+    },
+    "parser": "espree",
+
+    "extends": [
+        "eslint:recommended",
+        "google"
```

Comment:
What google extension for?

Reply:
an extention of "
    "eslint-config-google": "^0.14.0""